src/Power.cpp

```diff
@@ -668,8 +668,8 @@ void Power::readPowerStatus()
                 running++;
             }
         }
-        LOG_DEBUG("\n");
-        LOG_DEBUG("Heap status: %d/%d bytes free (%d), running %d/%d threads\n", memGet.getFreeHeap(), memGet.getHeapSize(),
+        LOG_DEBUG("");
```

Comment:
There's LOG_DEBUGs above this with no \n, probably assuming stuff is on the same line.